Tasks/ServiceFabricUpdateManifests/Update-Manifests.ps1

```diff
@@ -0,0 +1,26 @@
+# For more information on the VSTS Task SDK:
+# https://github.com/Microsoft/vsts-task-lib
+
+[CmdletBinding()]
+param()
+
+Trace-VstsEnteringInvocation $MyInvocation
+try {
+    # Import the localized strings.
+    Import-VstsLocStrings "$PSScriptRoot\task.json"
+
+    # Import helper modules
+    Find-VstsFiles -LiteralDirectory $PSScriptRoot -LegacyPattern "*.psm1" | ForEach { Import-Module $_ }
+
+    $updateType = (Get-VstsInput -Name updateType -Require)
```

Comment:
Well, you could do that but I don't think that leads to a better user experience.  You may want version updates with no Docker updates.  Or you may want Docker updates with no version updates.  So that means we need to allow the user to enable or disable both of those update types.  Which could lead to a configuration where neither are enabled.  Plus, the amount of settings would get cumbersome.  It seems better to divide things into smaller chunks.  This is a similar pattern to other tasks such as Docker.

---
In reply to: [136406059](https://github.com/Microsoft/vsts-tasks/pull/5076#discussion_r136406059) [](ancestors = 136406059)